config/manager/kustomization.yaml

```diff
@@ -2,4 +2,3 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - manager.yaml
-- credentials.yaml
```

Comment:
@andrewsykim -  note: this removes it also from `providers-components.yaml`,  If we want to remove the credentials only from `infrastructure-components.yaml` I can add them back using `generate.sh`